src/main/java/org/sigmah/server/dao/impl/GlobalExportHibernateDAO.java

```diff
@@ -27,6 +27,7 @@
 
 import javax.persistence.TypedQuery;
 
+import org.mortbay.log.Log;
```

Comment:
It would better if you replicate the log mechanism used in other part of the codebase.